pkg/storkctl/clusterpair.go

```diff
@@ -283,7 +290,12 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				return
 			}
 
-			if mode == "sync-dr" {
+			if mode != userInputCPModeAsync && mode != userInputCPModeSync && mode != userInputCPModeMigration {
+				util.CheckErr(fmt.Errorf("invalid mode %s, mode value should either be %s, %s or %s", mode, userInputCPModeAsync, userInputCPModeSync, userInputCPModeMigration))
```

Comment:
We will need to update the docs saying that now we need to pass on the mode explicitly? Can you file a PD for this?

Reply:
Till now it is not explicitly required, the default value for user input mode is `async-dr`.